apps/console/src/features/core/utils/user-store-utils.ts

```diff
@@ -66,4 +67,39 @@ export class SharedUserStoreUtils {
         const regEx = new RegExp(regExValue);
         return regEx.test(inputValue);
     };
+
+    /**
+     * The following method fetch the user store ids list.
+     */
+    public static getUserStoreIds = async (): Promise<string[]> => {
+        const userStoreIds: string[] = [];
+        await getUserStoreList().then((response) => {
+            response.data.map((userStore) => {
+                userStoreIds.push(userStore.id);
+            })
+        });
+
+        return userStoreIds;
+    };
```

Comment:
Any particular reason for using `await` here? Why shouldn't we go with something like this?
```suggestion
    public static async getUserStoreIds(): Promise<string[]> => {
        const userStoreIds: string[] = [];
        
        return getUserStoreList().then((response) => {
            response.data.map((userStore) => {
                userStoreIds.push(userStore.id);
            });
            
            return userStoreIds;
        });
    };
```

Reply:
Resolved with 3406d46